node/pkg/alephium/watcher.go

```diff
@@ -119,13 +120,22 @@ func NewAlephiumWatcher(
 		msgChan:   messageEvents,
 		obsvReqC:  obsvReqC,
 
-		pollInterval: pollInterval,
+		blockPollerEnabled: &atomic.Bool{},
+		pollIntervalMs:     pollIntervalMs,
 
 		client: NewClient(url, apiKey, 10),
 	}
 	return watcher, nil
 }
 
+func (w *Watcher) DisableBlockPoller() {
+	w.blockPollerEnabled.Store(false)
```

Comment:
If there are 2 poll requests, the block poller will be enabled. Once one of it is end, the flag will be set false. Will the second poll request be terminated too?

Reply:
No, the poll request is not for each individual transaction. It continues to poll the block height as long as there are pending transactions and only stops polling the block height when there are no pending transactions.

Reply:
perfect